examples/ui/text.rs

```diff
@@ -34,9 +34,8 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
             // Accepts a `String` or any type that converts into a `String`, such as `&str`
             "hello\nbevy!",
             TextStyle {
-                // This font is loaded and will be used instead of the default font.
                 font: asset_server.load("fonts/FiraSans-Bold.ttf"),
-                font_size: 100.0,
+                font_size: 100.0 / 1.2,
```

Comment:
IMO it's fine if we change how the example looks: these numbers are arbitrary anyways.